NYPLAudiobookToolkit/DRM/FeedbookDRMProcessor.swift

```diff
@@ -74,8 +74,8 @@ class FeedbookDRMProcessor {
      - Returns: Bool representing if the signature is valid
      */
     class private func verifySignature(_ signatureValue: String, forLicenseDoc: [String: Any]) -> Bool {
-        guard let privateKeyData = getFeedbookPrivateKeyFromKeychain(forVendor: "cantook") else {
-            ATLog(.error, "Private key for Feedbook is not found")
+        guard let publicKeyData = getFeedbookPrivateKeyFromKeychain(forVendor: "cantook") else {
```

Comment:
We should also change the method name from `private` to `public` to avoid confusion

not important: `FeedbookDRMPrivateKeyTag` too, but that would require change on SimplyE side

Reply:
I think we can create a new FeedbookDRMPublicKeyTag, with constant value reading "public" as well, and once the similar change is done in SimplyE, remove it from NYPLAudiobookToolkit

Reply:
Sure, sounds good to me